code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -562,3 +582,25 @@
 	Initialize()
 		. = ..()
 		reagents.add_reagent("sarezhiwine", 100)
+
+// Synnono Meme (Bottled) Drinks
+//======================================
+//
+
+/obj/item/weapon/reagent_containers/food/drinks/bottle/boukha
+	name = "Boukha Boboksa Classic"
+	desc = "A distillation of figs, imported from the Serene Republic of Elyra. Makes an excellent apertif or digestif."
+	icon_state = "boukhabottle"
+	center_of_mass = list("x"=16, "y"=6)
+	Initialize()
+		. = ..()
+		reagents.add_reagent("boukha", 100)
+
+/obj/item/weapon/reagent_containers/food/drinks/bottle/wwine

```

Comment:
Not a fan of this typepath, it's pretty similar to regular wine. Maybe `whitewine` instead?

Reply:
No issue with changing it. Was not thinking it would be an issue when compared to some of the other things on there, like icetea versus icedtea.

Standby.

Reply:
Those others should probably be changed _eventually_, though I'm not going to ask you to change them here.